.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Beta Feature Requirements
+about: Requirements for promoting a feature to beta
+---
+
+This page lists the requirements for promoting a feature to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Design doc has been approved
+
+**Config**
+
+- [ ] Can be enabled by default without requiring explicit user action. 
+
+**Docs** 
+
+- [ ] Performance expectations are documented, may have caveats. 
```

Comment:
Maybe rephrase like the following ones:

[ ] Documentation on istio.io includes performance expectations, may have caveats.

Reply:
Added. Thanks.